README.md

```diff
@@ -1,12 +1,48 @@
 # UA Parser - Browser Extension
 [![All Contributors](https://img.shields.io/badge/all_contributors-3-orange.svg?style=flat-square)](#contributors)
 
+## Prerequisites
+
+1. **NodeJS:** <br>
+   Please install [NodeJS >= 10.15.0](https://nodejs.org/en/download/). If you already have it, you're good to go.
+   <br>
+1. **Yarn:** <br>
```

Comment:
can we explicitly mention as list item `2` ?

Reply:
change according to the requirement.